components/d2l-organization-availability-set/lang/da.js

```diff
@@ -0,0 +1,13 @@
+/* eslint quotes: 0 */
+
+export default {
+	"addOrgUnits": "Tilføj organisationsenheder", // Add Org Units button
+	"currentOrgUnitItemDescription": "Aktuel organisationsenhed: {name}", // Label for a current orgunit availability
+	"explicitItemDescription": "{type}: {name}", // Label for an explicit orgunit availability with org unit type and name
+	"inheritItemDescription": "Hver organisationsenhed under {type}: {name}", // Label for an inherit (all descendants) orgunit availability
+	"inheritItemWithDescendantTypeDescription": "Hver {descendantType} under {type}: {name}", // Label for an inherit with specific {descendantType} orgunit availability
+	"removeAvailabilityFor": "Fjern tilgængelighed for {itemDescription}", // Alt text for remove availability 'X' button
+	"availabilityRemoved": "Tilgængelighed fjernet for {ItemDescription}", // Aria announcement when an availability is removed
```

Comment:
```suggestion
	"availabilityRemoved": "Tilgængelighed fjernet for {itemDescription}", // Aria announcement when an availability is removed
```
Translators should not have changed this